src/Time/Period.php

```diff
@@ -29,6 +29,31 @@ public function getEnd()
     return $this->endDate;
   }
 
+  public function inSeconds(): int
+  {
+    return $this->startDate->diffInSeconds($this->endDate);
+  }
+
+  public function inMinutes(): float
+  {
+    return round($this->inSeconds() / 60, 2);
```

Comment:
heb je round echt nodig? naja kan logisch zijn om te doen want als je vraagt of dagen, bv, ben je niet geintereseerd in 5 getallen achter de komma. Maar mogelijk moeten we dit veranderen.